people.json

```diff
@@ -976,6 +975,24 @@
       ],
       "image": "alessandro-vozza.jpeg"
    },
+   {
+      "name": "Alex Kattathra Johnson",
+      "bio": "<p>Multifaceted Engineer, who's curious about learning new technologies. Been passionate about CNCF technologies for almost a decade now.  First, as a software engineer at Cycle.io which is a LowOps containers as a service platform and then at Bombora (bombora.com) as a Devops Engineer supporting a MLE/DE team.  Also been passionate about being involved in CNCF open source projects as contributor.</p>",
+      "company": "",
+      "pronouns": "He/Him",
+      "location": "Bowling Green, United States",
```

Comment:
Could you change this to `Bowling Green, KY, United States` since there are multiple Bowling Greens in the nation?

Reply:
I noticed the state is missing in the location field of many of the newly added US kubestronauts.